src/sif/autocost.cc

```diff
@@ -454,8 +454,8 @@ bool AutoCost::AllowedReverse(const baldr::DirectedEdge* edge,
     return false;
   }
 
-  return DynamicCost::EvaluateRestrictions(access_mask_, edge, false, tile, opp_edgeid, current_time,
-                                           tz_index, restriction_idx);
+  return DynamicCost::EvaluateRestrictions(access_mask_, opp_edge, false, tile, opp_edgeid,
```

Comment:
There's a couple of these. Up until now, access restrictions were always symmetrical (meaning an edge's access restrictions were always equal to the ones of it's opposing edge), so it wasn't relevant that in `AllowedReverse`, the current edge's access restrictions were checked instead of the opposing one's.

Reply:
good catch :smile: